code/controllers/ExtensibleSearchAPI.php

```diff
@@ -93,22 +94,43 @@ public function toggleSuggestionApproved($request) {
 	 *	@URLparameter term <{SEARCH_TERM}> string
 	 *	@return JSON
 	 */
-
 	public function getSuggestions($request) {
 
-		if(Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions') && ($suggestions = $this->service->getSuggestions($request->getVar('term'), $request->getVar('page')))) {
+		if(Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions')) {
+			$suggestions = $this->service->getSuggestions($request->getVar('Search'), $request->getVar('page'));
```

Comment:
Why has this all been changed? It was originally returning a 404 when there were no search suggestions found (so the specific use case could be handled correctly), however it now looks to be returning an empty result set?